packages/electron-updater/src/MacUpdater.ts

```diff
@@ -95,6 +95,15 @@ export class MacUpdater extends AppUpdater {
     const updateFileSize = zipFileInfo.info.size ?? (await stat(downloadedFile)).size
 
     const log = this._logger
+
+    try {
+      this.debug("Clearing quarantine bit")
+      const result = execFileSync("xattr", ["-r", "-d", "com.apple.quarantine", downloadedFile])
+      log.info(`Quarantine bit is cleared: ${result}`)
```

Comment:
What's this `result` being printed out going to look like? Perhaps we could also append `?.trim()` in case there's extra whitespace

Reply:
in my case result was an empty string, perhaps it is always the case, need to check it.